chapter5/src/city.go

```diff
@@ -0,0 +1,27 @@
+package chapter5
```

Comment:
`package` name should not be `chapter5` but something that encapsulates the models or services you will define in the package. Also, when a package is imported, the package name becomes an accessor for the contents. Please read through [this](https://go.dev/doc/effective_go#package-names)

Reply:
Oh ok ok, thnx!